### DISCUSSION TITLE
JxlsException: Method createTransformer of org.jxls.transform.poi.PoiTransformer null

### DISCUSSION THREAD

**Discussion by daliadaud** [jxlsteam/jxls#264] [Q&A] [answered]
Hi, anybody still facing this issue ?
For context, I'm using 
jxls-poi 2.13.0
jxls 2.13

```
		<dependency>
			<groupId>org.jxls</groupId>
			<artifactId>jxls</artifactId>
			<version>2.13.0</version>
		</dependency>
		<dependency>
			<groupId>org.jxls</groupId>
			<artifactId>jxls-poi</artifactId>
			<version>2.13.0</version>
		</dependency>
```

Do we need to explicitly specify the poi in our pom.xml?
And the biggest issue is that the problem comes and goes each time we build the code

**Comment by daliadaud** [comment]
oddly enough the stack trace does not reflect the code base from 2.12 or 2.13 of PoiTransformer

stack trace
       at org.jxls.transform.poi.PoiTransformer.createTransformer(PoiTransformer.java:150)
        at org.jxls.transform.poi.PoiTransformer.createTransformer(PoiTransformer.java:141)
        at org.jxls.transform.poi.PoiTransformer.createTransformer(PoiTransformer.java:122)

createTransformer for 2.12 and 2.13 are at lines :
 
![image](https://github.com/jxlsteam/jxls/assets/13867522/34983baf-b988-4120-b52c-eeb52773fc1c)


is the version updated in maven?

**Comment by SoltauFintel** [comment, marked as answer]
Are Apache POI jar's in the class-path?
